bin/pint

```diff
@@ -67,10 +73,16 @@ options:
        Provide only SMT Server information
    --xml
        Output data in XML format
+   --bz2
+       Use Compression type bz2 to send compressed data over network
```

Comment:
add `(Default)` at the end as we will use bz2 compression if nothing is specified.